CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-<!--
+!--
```

Comment:
seems to be unexpected change, isn't it?

Reply:
Thank you